CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+  - [Built With](#built-with)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+
+<!-- ABOUT THE PROJECT -->
+
+## About The Project
+
+[Docusaurus](https://docusaurus.io/) is a static site generator that helps you ship beautiful, accessible docs. For building our documentation engine, we have made certain modifications to the template generated by [Docusaurus](https://docusaurus.io) to be able to properly showcase all our contents with their respective versions.
+
+### Built With
+
+- [Docusaurus](https://docusaurus.io/)
+- [React](https://reactjs.org/)
+- [Tailwind](https://tailwindcss.com/)
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+This section describes how you can get our documentation portal up and running on your machine.
+
+### Prerequisites
+
+- [node](https://nodejs.org/en/)
+- [npm](https://www.npmjs.com/)
+
+### Installation
+
+1. Clone the repo
```

Comment:
So will docusaurus be in a repo separate from the rest of the doc source?  For open source projects, we usually fork the repo then clone the fork, too.

Reply:
I have updated the  `CONTRIBUTING.md`  by adding fork detail too.